Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3860,27 +4116,39 @@ class Domain(Indicator):
         """
         CONTEXT_PATH = 'Domain(val.Name && val.Name == obj.Name)'
 
-        def __init__(self, domain, dbot_score, dns=None, detection_engines=None, positive_detections=None,
-                     organization=None, sub_domains=None, creation_date=None, updated_date=None, expiration_date=None,
-                     domain_status=None, name_servers=None, feed_related_indicators=None, malware_family=None,
-                     registrar_name=None, registrar_abuse_email=None, registrar_abuse_phone=None,
-                     registrant_name=None, registrant_email=None, registrant_phone=None, registrant_country=None,
-                     admin_name=None, admin_email=None, admin_phone=None, admin_country=None, tags=None,
-                     domain_idn_name=None, port=None,
-                     internal=None, category=None, campaign=None, traffic_light_protocol=None, threat_types=None,
-                     community_notes=None, publications=None, geo_location=None, geo_country=None,
-                     geo_description=None, tech_country=None, tech_name=None, tech_email=None, tech_organization=None,
-                     billing=None, relationships=None):
+        def __init__(self, domain, dbot_score, blocked=None, certificates=None, dns_records=None, dns=None,
```

Comment:
keep the original order of parameters (add new parameters to the end)

Reply:
Done